packages/bot-web-ui/src/components/dashboard/dashboard-components/sidebar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import Translations from './translations';
+import { localize } from '@deriv/translations';
+
+const { IntroSideBarContentText } = Translations;
+
+interface SideBarProps {
+    checkIfSidebarOpen: boolean;
+    setSideBarState: (state: boolean) => void;
+}
+
+const Sidebar = (props: SideBarProps) => {
+    const { setSideBarState, checkIfSidebarOpen } = props;
+    const sidebarClass = 'side_bar_container ';
+    const sidebarClasstoggle = checkIfSidebarOpen ? 'side_bar_container_block' : 'side_bar_container_none';
+
+    return (
+        <div className={sidebarClass + sidebarClasstoggle}>
+            <div className='side_bar_container_close_sidebar'>
+                <Icon
+                    className='dc-tabs__content_group_tiles_images'
+                    width='1rem'
+                    height='1rem'
+                    style={{ backgroundColor: `#F2F3F4` }}
```

Comment:
- Better to remove inline styles 
You can update the icon itself to cater for the this color.
`--general-section-1` will do the same and handle dark theme

Reply:
done